packages/augur-sdk/src/constants.ts

```diff
@@ -146,3 +146,13 @@ export const SECONDS_IN_AN_HOUR = new BigNumber(3600, 10);
 export const SECONDS_IN_A_DAY = new BigNumber(86400, 10);
 
 export const SECONDS_IN_A_YEAR = new BigNumber(SECONDS_IN_A_DAY).multipliedBy(365);
+
+export enum MarketReportingState {
```

Comment:
@bthaile The following MarketReporting states have been removed from this PR.
Forking = 'Forking'
AwaitingNoReportMigration = 'AwaitingNoReportMigration'
AwaitingForkMigration = 'AwaitingForkMigration'

We will need to update the UI accordingly starting here with packages/augur-ui/src/modules/common/constants.ts

Reply:
Is there a way to share these enums?